docs/diagrams/Formatter.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
```

Comment:
Guys for this class as well your file storage class I think it would better to make the class members private instead of public, because if I'm not wrong the it's better practice to have all class member have a private access modifier.